package.json

```diff
@@ -22,7 +22,8 @@
     "cloudfront:build:deploy:nc": "npm run client:build:deploy:nc && npm run cloudfront:invalidateCache",
     "cloudfront:update:build:deploy": "npm run cloudfront:setup && npm run cloudfront:build:deploy",
     "cloudfront:update:build:deploy:nc": "npm run cloudfront:setup && npm run cloudfront:build:deploy:nc",
-    "serverless:remove": "sls remove"
+    "serverless:remove": "sls remove",
+    "deploy": "az storage blob upload-batch -s './dist' -d '$web' --account-name 'storageaccountfrontend'"
```

Comment:
Let's make this "deploy" command less error-prone.
- What occurs if the `dist` folder goes missing?
- What happens if this command is executed a second time while the application is already deployed in storage?